SocialCareCaseViewerApi.Tests/V1/Helpers/Domain/CaseStatusDomainHelper.cs

```diff
@@ -0,0 +1,53 @@
+using Bogus;
+using SocialCareCaseViewerApi.V1.Domain;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SocialCareCaseViewerApi.Tests.V1.Helpers.Domain
+{
+    public static class CaseStatusDomainHelper
+    {
+        public static CaseStatusType NewCaseStatusType()
+        {
+            return new Faker<CaseStatusType>()
+                .RuleFor(cst => cst.Name, f => f.Lorem.Word())
+                .RuleFor(cst => cst.Description, f => f.Lorem.Sentence())
```

Comment:
We've had issues before with faker causing flaky tests when we don't explicitly state the strength length.

For instance using f.Person.Name instead of f.Random.String2(30). If the string chosen is too long it may not be saved to the database and a test will fail only some of the time.